config/toml.go

```diff
@@ -372,6 +361,18 @@ func ResetTestRootWithChainID(testName string, chainID string) *Config {
 	return config
 }
 
+func mustCreateTestRootDir(testName, chainID string) string {
```

Comment:
why extract this code into a function? ResetTestRootWithChainID does exactly that. see my comment above about creating a directory twice (or rather attempting to cause cmn.Ensure won't do it if dir already exist)

Reply:
reverted